cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ClusterCa.java

```diff
@@ -198,6 +199,7 @@ protected Map<String, CertAndKey> generateZkCerts(
             subject.addDnsName(zkDnsGenerator.wildcardServiceDnsName());
             subject.addDnsName(zkHeadlessDnsGenerator.wildcardServiceDnsNameWithoutClusterDomain());
             subject.addDnsName(zkHeadlessDnsGenerator.wildcardServiceDnsName());
+            subject.addDnsName(node.podName());
```

Comment:
Maybe you can move it somewhere up top to have it sorted from simplest to most complex? E.g. above the two pod names with more complete DNS addresses (i.e. above `subject.addDnsName(DnsNameGenerator.podDnsName(namespace, KafkaResources.zookeeperHeadlessServiceName(crName), node.podName()));`)?

Reply:
Yeah makes sense.